client/src/components/ui/SignUpModal.tsx

```diff
@@ -0,0 +1,138 @@
+import Image from "next/image";
+
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+function SignUpModal({ isOpen, onClose }: Props) {
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogTrigger asChild></DialogTrigger>
+      {/* Outer green container */}
+      <DialogContent
+        className="mx-auto flex h-full max-h-[725px] w-[95%] max-w-[600px] flex-col overflow-y-auto rounded-[40px] border-0 bg-[#9DAD93] p-[8px] shadow-lg"
+        style={{ backgroundColor: "#9DAD93", borderRadius: "40px" }}
```

Comment:
The style prop still needs to be removed, and use one of the colour variables defined in global.css